src/pages/Auth.css

```diff
@@ -0,0 +1,139 @@
+/* Page Container Styling */
+.page-container {
+    display: flex;
+    flex-direction: column; /* Stack elements vertically */
+    align-items: center;    /* Center content horizontally */
+    justify-content: flex-start; /* Start content from the top */
+    min-height: 100vh;      /* Take full viewport height */
+    background-color: #FAF5F2; /* Background color */
+    padding-top: 300px;     /* Add space at the top */
+}
```

Comment:
This affects other pages, can you update that `padding-top: 300px;`

Reply:
its just because the inbox page shares same class naming. updated to page-container-auth-page